pkg/api/scylla/v1alpha1/types_monitoring.go

```diff
@@ -162,6 +162,15 @@ type ScyllaDBMonitoringSpec struct {
 
 	// components hold additional config for the monitoring components in use.
 	Components *Components `json:"components"`
+
+	// type determines the platform type of the monitoring setup. Can be "Platform" or "SaaS".
+	// "Platform" sets up a detailed ScyllaDB monitoring, including the infrastructure.
+	// "Saas" deploys monitoring setup focusing on the ScyllaDB service.
+	// Defaults to "SaaS".
+	// +kubebuilder:validation:Enum="SaaS";"Platform"
```

Comment:
Since enumeration values are part of the API, shouldn't these be declared as consts in api package instead in `naming`?

Reply:
good point, changed